.gitignore

```diff
@@ -1 +1,5 @@
 node_modules/
+.cache
+.idea
+.vs
+dist
```

Comment:
dist shouldn't be in gitignore please remove that

Reply:
But dist is created every time we run the project. It is not recommended to exclude it from git-ignore. Also there will be be a long dist files to be committed together with every new commit if it is not ignored. What we can do is to add `/dist` to make sure only top level dist is ignored.

Reply:
Dist is where the code for `sbuttons.min.css` resides in, which we need. Do you have any other idea of where it should be?

Reply:
I see. No i just knew that it should be in .git-ignore. But in our case i see its not recommended.